portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1460,12 +1513,12 @@ definition {
 		AssertElementPresent(
 			key_inputMaskValue = "${inputMaskValue}",
 			locator1 = "FormFields#INPUT_MASK_VALUE");
-	}
 
-	macro viewNumericType {
-		AssertElementPresent(
-			key_numericType = "${numericType}",
-			locator1 = "FormFields#NUMERIC_TYPE");
```

Comment:
@rafaellajordao, [your pull](https://github.com/rafaellajordao/liferay-portal/commit/aca6837cc224e1ec876869c5bc966b8610a87a34) will be affected by this change, can you please update your pull according it. Please talk to @leticiachiu if you think this is necessary to avoid code conflicts. :slightly_smiling_face: 

Thanks